python/cudf/cudf/_lib/binops.pyx

```diff
@@ -113,6 +113,16 @@ def apply_op(lhs, rhs, out, op):
             <gdf_column*> c_out,
             op
         )
+    elif lhs is None:
```

Comment:
Future extension: Support more than just `None` as supported `null` scalars here. PyArrow has a concept of a scalar `null`, and we'll likely create our own for cuDF.